### PR TITLE
Allow buttons to take null value

### PR DESCRIPTION
**Pull request by abidlabs**
Fixes: https://github.com/gradio-app/gradio/issues/6869

Test code in original issue

### CODE REVIEW

File: js/utils/src/utils.ts [hunk 1]
```diff
@@ -151,11 +151,6 @@ async function copy_to_clipboard(value: string): Promise<boolean> {
 	return copied;
 }
 
-// import { format } from "svelte-i18n";
```
Comment (freddyaboulton) [inline js/utils/src/utils.ts:154]: Is this intentional?

**Review by freddyaboulton — APPROVED** [review]
Looks good @abidlabs ! Can we unit test this?

File: js/utils/src/utils.ts (hunk 1 above)
Comment (abidlabs) [reply js/utils/src/utils.ts:154]: it seemed like an unused comment so I removed it. am I missing something? cc @hannahblair just in case

**Review by pngwn — APPROVED** [review]
Looks great! Thanks @abidlabs!

**Comment by abidlabs** [comment]
Thanks folks for the review! I added a story to catch regressions.

**Pull request merged**